lib/pages/dashboard/library_screen.dart

```diff
@@ -130,23 +133,28 @@ class LibraryScreen extends StatelessWidget {
                       ),
                       delegate: SliverChildBuilderDelegate(
                         (BuildContext context, int index) {
-                          return LibraryGridTile();
+                          return LibraryGridTile(
+                            image: recentPlayedItems[index].image,
```

Comment:
Usually it is better to pass models rather than individual fields when there are multiple params. Upto 2 should be fine but assess the fact that the actions in grid tile might need more than 2 parameters and change accordingly. Or u can leave it be for now and make all future changes with this in mind